.azure/infrastructure/main.bicep

```
@@ -1,20 +1,32 @@
 targetScope = 'subscription'
+
+@description('The environment for the deployment')
 @minLength(3)
 param environment string
+
+@description('The location where the resources will be deployed')
 @minLength(3)
 param location string
 
+@description('Array of all keys in the source Key Vault')
 param keyVaultSourceKeys array
 
+@description('Password for postgres admin')
```

```suggestion
@description('Password for PostgreSQL admin')
```

🤷🏼 If you want consistency with the other `PostgreSQL`s